tests/unit/ir/node_data.cpp

```diff
@@ -41,6 +41,22 @@ int main(int argc, char* argv[])
         test_serialization(single_value);
     }
 
+    {
+        phylanx::ir::node_data<bool> single_value(false);
+        HPX_TEST_EQ(single_value[0], false);
+
+        //auto begin = hpx::util::begin(single_value);
+        //auto end = hpx::util::end(single_value);
+        //
+        //HPX_TEST(begin != end);
+        //HPX_TEST_EQ(*begin, false);
+        //HPX_TEST(++begin == end);
```

Comment:
I think we can remove the commented code now as we have dropped `node_data::begin/end`.